acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -7,7 +7,7 @@
 test_name "Install Packages"
 
 step "Install repositories on target machines..." do
-  sha = ENV['SHA']
+  sha = ENV['VERSION'] || ENV['SHA']
```

Comment:
I'm wondering if this will have an implications in CI? If the job defines `VERSION` environment variable to be something other than `SHA`, then acceptance will likely fail. Also metaquestion, is `VERSION` the preferred way of specifying the "thing" to test?